documentation/asciidoc/computers/os/updating.adoc

```diff
@@ -120,13 +120,13 @@ sudo apt purge tree
 
 WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process.
 
-The `rpi-update` script is supplied by a third party, "Hexxeh", and also supported by Raspberry Pi engineers. The script source can be found on GitHub at https://github.com/Hexxeh/rpi-update
+The `rpi-update` script (originally written by https://github.com/Hexxeh[Hexxeh]) is supported by Raspberry Pi engineers. The script source is in the https://github.com/raspberrypi/rpi-update[rpi-update repository]
```

Comment:
I wonder if the first sentence would read better as "The `rpi-update` script was originally written by [Hexxeh](https://github.com/Hexxeh), but is now supported by Raspberry Pi engineers." ?
And could you add a full-stop at the end of the second sentence please?

Reply:
Definitely better.